test/test_cftime.py

```diff
@@ -686,10 +686,6 @@ def __init__(self, start, n, step, units, calendar='standard'):
             and `step` define the starting date, the length of the array and
             the distance between each date (in units).
 
-            :Example:
-            >>> t = TestTime(datetime(1989, 2, 18), 45, 6, 'hours since 1979-01-01')
-            >>> print num2date(t[1], t.units)
-            1989-02-18 06:00:00
```

Comment:
This test doc-test is failing after I enabled runnable doc-tests, took the decision to remove it... hope that's okay?